test/administrative-sdkSpec.js

```diff
@@ -65,7 +65,7 @@ describe('Administrative SDK', () => {
     sdk.getPronunciationChallenge(1, 2);
     sdk.listPronunciationChallenges(1);
     sdk.deletePronunciationChallenge(1);
-    sdk.createSpeechChallenge(1);
+    sdk.createSpeechChallenge(1, 2);
```

Comment:
Yeah I guess. These tests are more to just see if the calls get redirected properly with the arguments.